src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,54 @@
 'use strict';
 
+const damage = 50;
```

Comment:
Good, but in this case it is better to make the constant a part of the `Carnivore` class